app/assets/javascripts/autoform_form.js

```diff
@@ -29,33 +44,19 @@ $(document).ready(function() {
             return true;
         }
     });
-
-    var autoformError = $('#autoform-error').get(0);
```

Comment:
@jsuchal konkretne tato cast, ktora hladala na kazdej pagy autoform classy

Reply:
Ale ved to by nemalo vadit nie? Nenajde tak nie je problem.